trieste/models/gpflow/builders.py

```diff
@@ -423,3 +424,44 @@ def _get_inducing_points(
         inducing_points = search_space.sample(num_inducing_points)
 
     return inducing_points
+
+
+def build_ar1_models(
+    dataset: Dataset, num_fidelities: int, input_search_space: SearchSpace
+) -> Sequence[GaussianProcessRegression]:
+    """
+    Build the individual GPR models required for constructing an AR1 model
+    with `num_fidelities` fidelities.
+
+    :param dataset: Dataset of points with which to initialise the individual models,
+        where the final column of the final dimension of the query points contains the fidelity
+    :param num_fidelities: Number of fidelities desired for the AR1 model
+    :param input_search_space: The input search space of the models
+    :return: List of initialised GPR models
+    """
+    if num_fidelities < 2:
+        raise ValueError(
+            "Invalid number of fidelities to build AR1 model for,"
+            f" need at least 2 fidelities, got {num_fidelities}"
+        )
+
+    # Split data into fidelities
+    data = split_dataset_by_fidelity(dataset=dataset, num_fidelities=num_fidelities)
+
+    for i, fidelity_data in enumerate(data):
+        if len(fidelity_data) < 2:
+            raise ValueError(
+                f"Not enough data to create model for fidelity {i},"
+                f" need at least 2 datapoints, got {len(fidelity_data)}"
+            )
+
+    gprs = [
+        GaussianProcessRegression(
+            build_gpr(
+                data[fidelity], input_search_space, likelihood_variance=1e-6, kernel_priors=False
```

Comment:
does it make sense to make `likelihood_variance` and `kernel_priors` optional inputs?